client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/provision-status/test/index.tsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * @jest-environment jsdom
+ */
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { ProvisionStatus } from '..';
+
+describe( 'ProvisionStatus', () => {
+	it( 'should render the first non-successful action', () => {
+		const status = {
+			siteTransfer: 'success',
+			pluginInstallation: 'pending',
+			migrationKey: 'error',
+		};
+
+		render( <ProvisionStatus status={ status } /> );
+
+		expect( screen.getByText( 'Installing the required plugins' ) ).toBeVisible();
+	} );
+
+	it( 'should render the correct action progress number', () => {
+		const status = {
+			siteTransfer: 'success',
+			pluginInstallation: 'pending',
+			migrationKey: 'error',
+		};
+
+		render( <ProvisionStatus status={ status } /> );
+
+		expect( screen.getByText( '2/3' ) ).toBeVisible();
+	} );
+
+	it( 'should render success message when all actions are successul', () => {
+		const status = {
+			siteTransfer: 'success',
+			pluginInstallation: 'success',
+			migrationKey: 'success',
+		};
+
+		render( <ProvisionStatus status={ status } /> );
+
+		expect(
+			screen.getByText(
+				/Your new site is ready! Enter your migration key into your old site to start your migration./
+			)
+		).toBeVisible();
+	} );
+
+	it( 'should render alternative success message when plugins installation is done, but migration key is not available anymore', () => {
```

Comment:
```suggestion
	it( 'shows instructions to the user get the key by itself when we can get the key", () => {
```

Reply:
Same idea I explained on the previous comment

Reply:
Just did, but actually used **can't** on `can't get the key`.
I assume that was your intention: When we can't get the key through the API, shows instructions to the user get the key by itself.